content/projects/tdd/string-calculator-part-1/_index.md

```diff
@@ -166,13 +166,13 @@ In the first case, the default delimiter is: ";".
 
 In the second case, the default delimiter is: "4".
 
-**Note:** you are not expected to handle integer delimiters where the delimiter and the digit you are adding are next to each other. Example:
+**Note:** you are not expected to handle integer delimiters where the delimiter and the digit you are adding are identical and next to each other. Example:
```

Comment:
@Lindo-code this part can confuse learners don't you think?

- Our sentence in 169 will make sense when the add function is called this way : `add("//8\n1882")`
- In the example in line 172, "88" is technically not identical to "8".
- I agree that the old sentence is confusing, just think more on how to change it an a manner, where it makes sense. there are atleast two edge cases to consider here, make sure your new sentence satisfies them both.